go.mod

```diff
@@ -103,4 +103,6 @@ replace (
 
 	// until merged upstream: https://github.com/mwitkow/grpc-proxy/pull/69
 	github.com/mwitkow/grpc-proxy => github.com/smartcontractkit/grpc-proxy v0.0.0-20230731113816-f1be6620749f
+
+	github.com/smartcontractkit/chainlink-relay => github.com/smartcontractkit/chainlink-relay v0.0.0-20231101034200-60ac98bbd672
```

Comment:
If you just use the version directly, then it will be inherited automatically, and you won't have to spam this replace into every go.mod file.